metalog.js

```diff
@@ -88,6 +88,14 @@ class Logger extends EventEmitter {
     this.lock = false;
     this.buffer = [];
     this.file = '';
+    if (!fs.existsSync(path)) {
```

Comment:
Don't use blocking functions for checking is file exists.
Also don't use `fs.exists` because it is deprecated, see: https://nodejs.org/api/fs.html#fs_fs_exists_path_callback
We need `fs.access` here, see: https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback
But I think we need to move this check to `open` method and it's better to add `async` keyword to this method.
We have different issue for that: https://github.com/metarhia/metalog/issues/106
So it's better to implement #106 and after that rewrite #107

Reply:
Should I close this PR and make a new one to implement issue #106? I think, if I will make new commits, the changes will land here. Or maybe is there more graceful decision?

Reply:
No, just open new PR for those task and after landing it rebase this to master and resolve conflicts.